docs/application/web/api/6.5/device_api/mobile/tizen/ml_pipeline.html

```diff
@@ -0,0 +1,2120 @@
+            </div>
+<p><span class="version">Since: </span>
+ 6.5
+            </p>
+</li>
+</ul>
+</div>
+<div class="methods">
+<h4>Methods</h4>
+<dl>
+<dt class="method" id="Switch::getPadList">
+<a class="backward-compatibility-anchor" name="::Pipeline::Switch::getPadList"></a><code><b><span class="methodName">getPadList</span></b></code>
+</dt>
+<dd>
+<div class="brief">
+ Retrieves the list of pad names of the switch.
+            </div>
+<div class="synopsis"><pre class="signature prettyprint">DOMString[] getPadList();</pre></div>
+<p><span class="version">Since: </span>
+ 6.5
+            </p>
+<div class="returntype">
+<p><span class="return">Return value:</span></p>
+<ul>
+<span class="return">DOMString[]:</span>
+ padList The list of switch's pads.
+              </ul>
+</div>
+<div class="exceptionlist">
+<p><span class="except">Exceptions:</span></p>
+          <ul class="exception"><li>WebAPIException<ul>
+<li class="list"><p>
+ with error type NotSupportedError, if the feature is not supported.
+                </p></li>
+<li class="list"><p>
+ with error type NotFoundError, if the switch belonged to a disposed pipeline.
+                </p></li>
+<li class="list"><p>
+ with error type InvalidStateError, if the pipeline is not ready yet.
+                </p></li>
+<li class="list"><p>
+ with error type AbortError, if any other error occurs.
+                </p></li>
+</ul>
+</li></ul>
+        </div>
+<div class="example">
+<span class="example"><p>Code example:</p></span><pre name="code" class="examplecode prettyprint">var pipelineDescription = "videotestsrc is-live=true " +
+                          "! videoconvert " +
+                          "! tensor_converter " +
+                          "! output-selector name=outs outs.src_0 " +
+                          "! tensor_sink name=sink0 async=false outs.src_1 " +
+                          "! tensor_sink name=sink1 async=false";
+
+var pipeline = tizen.ml.pipeline.createPipeline(pipelineDescription);
+
+var outsSwitch = pipeline.getSwitch("outs");
+var pads = outsSwitch.getPadList();
+
+console.log("Pads:");
+for (var i = 0; i &lt; pads.length; i++)
+{
+  console.log("  " + pads[i]);
+}
+</pre>
+</div>
+<div class="output">
+<span class="title"><p>Output example:</p></span><pre>Pads:
+  src_0
+  src_1
+</pre>
+</div>
+</dd>
+<dt class="method" id="Switch::select">
+<a class="backward-compatibility-anchor" name="::Pipeline::Switch::select"></a><code><b><span class="methodName">select</span></b></code>
+</dt>
+<dd>
+<div class="brief">
+ Selects a pad to be used as a source or sink of the switch node.
+            </div>
+<div class="synopsis"><pre class="signature prettyprint">void select(DOMString padName);</pre></div>
+<p><span class="version">Since: </span>
+ 6.5
+            </p>
+<div class="description">
+            <p>
+The switch selects a sink if its <a href="#Switch::type">Switch::type</a> is "OUTPUT_SELECTOR" or a source if it is "INPUT_SELECTOR".
```

Comment:
Nor clear, please rephrase.

Reply:
This is very confusing

Reply:
The switch selects a sink if its <a href="#Switch::type">Switch::type</a> is "OUTPUT_SELECTOR" or it selects a source if it is <a href="#Switch::type">Switch::type</a> is "INPUT_SELECTOR".